prettier.config.js

```diff
@@ -5,12 +5,10 @@ const prettierConfig = {
     trailingComma: 'es5',
 }
 
-module.exports = Object.assign({}, prettierConfig, {
-    overrides: [
-        {
-            files: 'package.json',
-            options: { tabWidth: 2, parser: 'json' },
-        },
-    ],
-    prettierConfig,
-})
+exports.prettierConfig = prettierConfig
```

Comment:
to use this config in other repos we need to export the `prettierConfig` as main export

Reply:
we may agree to not run prettier for `json` files and can remove the `overrides` part.
This would allow us to not use the `.prettierrc` and have the config duplicated.